### PR TITLE
Allows for Mobile users to click on link

### PR DESCRIPTION
**Pull request by mattdax**
We noticed that links were clickable on mobile phones, fixed a few lines and now it works.

Thanks for the great package!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "panzoom",
   "version": "2.5.0",
   "description": "Pan and zoom SVG elements",
-  "main": "index.js",
+  "main": "dist/panzoom.min.js",
```
Comment (anvaka) [inline package.json]: Why this change?

File: dist/panzoom.js [hunk 2]
```diff
@@ -348,7 +348,7 @@ function createPanZoom(svgElement, options) {
 
   function onTouch(e) {
     if (e.touches.length === 1) {
-      return handleSignleFingerTouch(e, e.touches[0])
+      return handleSingleFingerTouch(e, e.touches[0])
```
Comment (anvaka) [inline dist/panzoom.js]: Doh. Thanks for catching this!

File: package.json (hunk 1 above)
Comment (mattdax) [inline package.json]: minifies the code into to make it more efficient when rand on a website...Essentially compresses it

Why You Should Compress JavaScript
There are a number of reasons why compressing your JavaScript files is a good idea:
Quicker download times for your users.
Reduced bandwidth consumption of your website.
Reduced number of HTTP requests on your server when combining many JavaScript files into one compressed file, thus reducing the server load and allowing more visitors to access your website.
Comments and whitespace are not needed for JavaScript execution; Removing them will reduce file size and speed up script execution times.

Comment (anvaka) [inline package.json]: Yes, but by putting `dist/panzoom.min.js` into `main` clause we are loosing opportunity for other bundlers to inline this module with proper source maps. We are also loosing benefits of improved minification algorithms, that new library clients can decide to use in future.

**Comment by Tusky** [comment]
This would be great to be included, since on mobile the Taps are not registering, I can't find a way to make it work from my code. I'm using this with React, and click works from browser, but the tap is caught by panzoom. :(

**Comment by alextappin** [comment]
Any update on when this can be merged?

**Comment by rbonomo** [comment]
@Tusky Add an event listener on your element you want clickable. In the handler function, call event.stopPropagation() to prevent panzoom from capturing the event.

**Comment by Tusky** [comment]
Did you just necro me after more than a year? :D I'm not on the project using it, but I resolved by fixing the typo above. :) That's why I said back than this would be great if merged, even if partially, I'd imagine this project also grown since I've used it.

File: package.json (hunk 1 above)
Comment (jamesabad99) [reply package.json:5]: @mattdax hey man, i wonder from where can i pull the code, i pretty much a noob in coding, and i was wondering if can just pull the code like this with the fixes that you made, it would be lifesaver  <script> src='https://unpkg.com/panzoom@9.4.0/dist/panzoom.min.js' </script>

**Comment by jamesabad99** [comment]
> Did you just necro me after more than a year? :D I'm not on the project using it, but I resolved by fixing the typo above. :) That's why I said back than this would be great if merged, even if partially, I'd imagine this project also grown since I've used it.

@Tusky hey man, is there somewhere that i can pull the code from? like this  <script> src='https://unpkg.com/panzoom@9.4.0/dist/panzoom.min.js' </script>
